src/test/fixtures/ArcadiaVaultsFixture.f.sol

```diff
@@ -0,0 +1,355 @@
+/**
+ * Created by Arcadia Finance
+ * https://www.arcadia.finance
+ *
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity >0.8.10;
+
+import "../../../lib/forge-std/src/Test.sol";
+
+import "../../Factory.sol";
+import "../../Proxy.sol";
+import "../../Vault.sol";
+import {ERC20Mock} from "../../mockups/ERC20SolmateMock.sol";
+import "../../mockups/ERC721SolmateMock.sol";
+import "../../mockups/ERC1155SolmateMock.sol";
+import "../../AssetRegistry/MainRegistry.sol";
+import "../../AssetRegistry/FloorERC721PricingModule.sol";
+import "../../AssetRegistry/StandardERC20PricingModule.sol";
+import "../../AssetRegistry/FloorERC1155PricingModule.sol";
+import "../../Liquidator.sol";
+import "../../OracleHub.sol";
+import "../../utils/Constants.sol";
+import "../../mockups/ArcadiaOracle.sol";
+import ".././fixtures/ArcadiaOracleFixture.f.sol";
+
+contract DeployArcadiaVaults is Test {
+    Factory public factory;
+    Vault public vault;
+    Vault public proxy;
+    address public proxyAddr;
+    ERC20Mock public dai;
+    ERC20Mock public eth;
+    ERC20Mock public snx;
+    ERC20Mock public link;
+    ERC20Mock public safemoon;
+    ERC721Mock public bayc;
+    ERC721Mock public mayc;
+    ERC721Mock public dickButs;
+    ERC20Mock public wbayc;
+    ERC20Mock public wmayc;
+    ERC1155Mock public interleave;
+    OracleHub public oracleHub;
+    ArcadiaOracle public oracleDaiToUsd;
+    ArcadiaOracle public oracleEthToUsd;
+    ArcadiaOracle public oracleLinkToUsd;
+    ArcadiaOracle public oracleSnxToEth;
+    ArcadiaOracle public oracleWbaycToEth;
+    ArcadiaOracle public oracleWmaycToUsd;
+    ArcadiaOracle public oracleInterleaveToEth;
+    MainRegistry public mainRegistry;
+    StandardERC20PricingModule public standardERC20PricingModule;
+    FloorERC721PricingModule public floorERC721PricingModule;
+    FloorERC1155PricingModule public floorERC1155PricingModule;
```

Comment:
rationale between having these three here and not all (or one)?

Reply:
Many test files (Mainregistry, EndToEnd, Gastests) use those 3 and covers the different tokenstandards